assets/src/edit-story/components/library/panes/text/textSets/raw/section_header.json

```
@@ -716,17 +716,17 @@
             "vertical": 0
           },
           "type": "text",
-          "content": "The aesthetic quality of a product is integral to its usefulness because products we use every day affect our person and our well-being. But only well-executed objects can be beautiful.",
+          "content": "San Francisco is only seven by seven miles, but its packed of distinct neighborhoods. Stay in the Haight Asbury to taste the Summer of Love, or book off Columbus street for a culinary collision between Little Italy and Chinatown. ",
```

```suggestion
          "content": "San Francisco is only seven by seven miles, but it's packed of distinct neighborhoods. Stay in the Haight Asbury to taste the Summer of Love, or book off Columbus street for a culinary collision between Little Italy and Chinatown. ",
```

think this should be but it IS packed of distinct neighborhoods, so added an apostrophe